src/ivoc/strfun.cpp

```diff
@@ -39,15 +38,18 @@ static double l_len(void*) {
     return double(strlen(gargstr(1)));
 }
 
+static std::string imitate_multiline(const std::string& text) {  // Clean the text so we keep only
+                                                                 // the first line Imitation of
+                                                                 // std::multiline in our case
+    static const std::regex r(R"(^(.*)(\n|$))");
```

Comment:
I was a bit confused here at first, due to `r(R...)`, but then realized what you meant. I personally find `static const auto r = std::regex(...)` easier to parse (that there are 15 different ways to initialize variables in C++ definitely doesn't help). Just a style preference, not a big issue.